### PR TITLE
HV-953

### PR DESCRIPTION
**Pull request by hferentschik**
Integration test still missing (could potentially be provided later). Ready for review though.

### CODE REVIEW

**Comment by hferentschik** [comment]
Added wildfly integration test

File: engine/src/main/java/org/hibernate/validator/resourceloading/AggregateResourceBundle.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+import java.util.Properties;
+import java.util.ResourceBundle;
+
+/**
+ * Inspired by <a href="http://stackoverflow.com/questions/4614465/is-it-possible-to-include-resource-bundle-files-within-a-resource-bundle">this</a>
+ * Stack Overflow question.
+ *
+ * @author Hardy Ferentschik
+ */
+public class AggregateResourceBundle extends ResourceBundle {
```
Comment (gunnarmorling) [inline engine/src/main/java/org/hibernate/validator/resourceloading/AggregateResourceBundle.java]: Is there any difference to the inner class of the same name in `PlatformResourceBundleLocator`? Can this be merged to one class? If not, different names would be nice to make the different semantics apparent.

Comment (gunnarmorling) [inline engine/src/main/java/org/hibernate/validator/resourceloading/AggregateResourceBundle.java]: Interesting, this seems to be un-used. Maybe a left-over?

File: integration/src/test/java/org/hibernate/validator/integration/wildfly/ConstraintDefinitionContributorUnitIT.java [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+	private static final String WAR_FILE_NAME = ConstraintDefinitionContributorUnitIT.class.getSimpleName() + ".war";
+
+	@Inject
+	private Validator validator;
+
+
+	@Deployment
+	public static WebArchive createTestArchive() throws Exception {
+		return ShrinkWrap
+				.create( WebArchive.class, WAR_FILE_NAME )
```
Comment (gunnarmorling) [inline integration/src/test/java/org/hibernate/validator/integration/wildfly/ConstraintDefinitionContributorUnitIT.java]: Very nice test!

**Comment by gunnarmorling** [comment]
Rebased and applied. Added two more commits to remove that superfluous class and do some other minor fixes.

**Comment by hferentschik** [comment]
Thanks, good catch with the `AggregateResourceBundle`. I meant to delete it, but apparently I forgot.

**Pull request closed without merging**